core/src/com/rallyhealth/weepickle/v1/core/ValidatingVisitor.scala

```diff
@@ -0,0 +1,236 @@
+package com.rallyhealth.weepickle.v1.core
+
+import java.time.Instant
+
+import scala.annotation.tailrec
+import scala.collection.mutable.ListBuffer
+import scala.util.control.NonFatal
+import scala.util.{Failure, Success, Try}
+
+/**
+  * A slight variation of JsonPointerVisitor, where we
+  * add a *sequence* of JSON Pointers to the result of the delegate Visitor
+  * rather than throwing a single exception for the first error encountered by the delegate.
+  *
+  * JSON Pointer is standardized by RFC 6901 and commonly used by JSON Schema.
+  *
+  * Useful for debugging failures.
+  * Adds ~10% overhead depending on the parser. [?]
```

Comment:
How does this compare to JsonPointerVisitor?

Reply:
I did not benchmark. These comments were carried forward from JsonPointerVisitor.

Reply:
Here are some microbenchmarks against a 640 KB, 24 K line Json with two errors:
```
[info] Benchmark               Mode  Cnt    Score   Error  Units
[info] BenchTest.play_All     thrpt   20   57.468 ± 1.367  ops/s
[info] BenchTest.wee_1_First  thrpt   20  229.404 ± 5.225  ops/s
[info] BenchTest.wee_2_All    thrpt   20  184.232 ± 3.377  ops/s
```
So seems that it adds about 20% overhead